demo/chbench/dc.sh

```diff
@@ -309,16 +310,25 @@ clean_load_test() {
     load_test --up
 }
 
+# Purges Kafka topic and restarts Materialize/peeker
+drop_kafka_topics() {
+    dc_stop chbench
+    dc_stop materialized peeker
+    runv docker exec -it chbench_kafka_1 kafka-topics --delete --bootstrap-server localhost:9092 --topic "mysql.tpcch.*" || true
+    dc_up materialized
+}
+
 # Long-running load test
 load_test() {
     if [[ "${1:-}" = --up ]]; then
         initialize_warehouse
         bring_up_source_data
         bring_up_introspection
     fi
+    drop_kafka_topics
```

Comment:
I am unsure if this should go into an `else` branch. I think it's fine here, but it's complicated. If this is being run after `clean_test_load` it's never necessary, I don't think, but any other time it's broken.

I think this is fine and I should just collapse `clean_test_load` into this function.